vm-setup/Vagrantfile

```diff
@@ -23,7 +23,6 @@ Vagrant.configure("2") do |config|
   # within the machine from a port on the host machine. In the example below,
   # accessing "localhost:8080" will access port 80 on the guest machine.
   config.vm.network "forwarded_port", guest: 8443, host: 8443
-  config.vm.network "forwarded_port", guest: 80, host: 80
```

Comment:
Why is this not needed any more?

Reply:
It never worked, port 80 is a privileged port so unl;ess vb was running as a root it port forwarding was not enabled.